CHANGELOG.md

```diff
@@ -210,107 +54,11 @@ This is the easiest way to work with the released versions of PWA Studio.
 
 #### Upgrade method: Update `package.json`
 
-To upgrade to 6.0.0, update the project's `package.json` file and change the dependency version for PWA Studio.
+To upgrade to 6.0.1, update the project's `package.json` file and change the dependency version for PWA Studio.
```

Comment:
This sentence slightly confuses me. There's no single PWA Studio dependency, so this might be clearer if it said "change the version strings for any PWA Studio package dependencies".